Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -1121,4 +1114,275 @@ defineSuite([
+            expect(stats.numberOfCommands).toEqual(5);
+            expect(stats.numberReady).toEqual(5);
+
+            // Orient camera to face the sky
+            var center = Cartesian3.fromRadians(centerLongitude, centerLatitude, 100);
+            scene.camera.lookAt(center, new HeadingPitchRange(0.0, 1.57, 10.0));
+
+            // All tiles are unloaded
+            scene.renderForSpecs();
+            expect(stats.numberOfCommands).toEqual(0);
+            expect(stats.numberReady).toEqual(0);
+
+            // Reset camera so all tiles are reloaded
+            viewAllTiles();
+
+            return Cesium3DTilesTester.waitForPendingRequests(scene, tileset).then(function() {
+                scene.renderForSpecs();
+                expect(stats.numberOfCommands).toEqual(5);
+                expect(stats.numberReady).toEqual(5);
+            });
+        });
+    });
+
+    it('Unloads cached tiles in a tileset with external tileset.json', function() {
+        return Cesium3DTilesTester.loadTileset(scene, tilesetOfTilesetsUrl).then(function(tileset) {
+            var stats = tileset._statistics;
+            var replacementList = tileset._replacementList;
+
+            tileset.maximumNumberOfLoadedTiles = 2;
+
+            scene.renderForSpecs();
+            expect(stats.numberOfCommands).toEqual(5);
+            expect(stats.numberReady).toEqual(7); // 5 with b3dm content + 2 empty/tileset.json
+            expect(replacementList.length - 1).toEqual(5); // Only tiles with content are on the replacement list. -1 for sentinel.
+
+            // Zoom out so only root tile is needed to meet SSE.  This unloads
+            // all tiles except the root and one of the b3dm children
+            viewRootOnly();
+            scene.renderForSpecs();
+
+            expect(stats.numberOfCommands).toEqual(1);
+            expect(stats.numberReady).toEqual(4); // 2 with b3dm content + 2 empty/tileset.json
+            expect(replacementList.length - 1).toEqual(2);
+
+            // Reset camera so all tiles are reloaded
+            viewAllTiles();
+
+            return Cesium3DTilesTester.waitForPendingRequests(scene, tileset).then(function() {
+                scene.renderForSpecs();
+                expect(stats.numberOfCommands).toEqual(5);
+                expect(stats.numberReady).toEqual(7);
+                expect(replacementList.length - 1).toEqual(5);
+            });
+        });
+    });
+
+    it('Unloads cached tiles in a tileset with empty tiles', function() {
+        return Cesium3DTilesTester.loadTileset(scene, tilesetEmptyRootUrl).then(function(tileset) {
+            var stats = tileset._statistics;
+            var replacementList = tileset._replacementList;
+
+            tileset.maximumNumberOfLoadedTiles = 2;
+
+            scene.renderForSpecs();
+            expect(stats.numberOfCommands).toEqual(4);
+            expect(stats.numberReady).toEqual(4); // 4 children with b3dm content (does not include empty root)
```

Comment:
Because it is defined as:
> numberReady : 0, // Number of tiles with content loaded

It only accounts for tiles that come off the processing queue; empty tiles do not.